pkg/controller/statefulset/stateful_pod_control.go

```diff
@@ -207,15 +207,15 @@ func (spc *StatefulPodControl) DeleteStatefulPod(set *apps.StatefulSet, pod *v1.
 // ClaimsMatchRetentionPolicy returns false if the PVCs for pod are not consistent with set's PVC deletion policy.
 // An error is returned if something is not consistent. This is expected if the pod is being otherwise updated,
 // but a problem otherwise (see usage of this method in UpdateStatefulPod).
-func (spc *StatefulPodControl) ClaimsMatchRetentionPolicy(set *apps.StatefulSet, pod *v1.Pod) (bool, error) {
+func (spc *StatefulPodControl) ClaimsMatchRetentionPolicy(ctx context.Context, set *apps.StatefulSet, pod *v1.Pod) (bool, error) {
 	ordinal := getOrdinal(pod)
 	templates := set.Spec.VolumeClaimTemplates
 	for i := range templates {
 		claimName := getPersistentVolumeClaimName(set, &templates[i], ordinal)
 		claim, err := spc.objectMgr.GetClaim(set.Namespace, claimName)
 		switch {
 		case apierrors.IsNotFound(err):
-			klog.V(4).Infof("Expected claim %s missing, continuing to pick up in next iteration", claimName)
+			klog.FromContext(ctx).V(4).Info("Expected claim missing, continuing to pick up in next iteration", "claim", claimName)
```

Comment:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md mentions "PVC" as good key for a `PersistentVolumeClaim` object.

Please replace ` "claim", claimName` with `"PVC", klog.KRef(..., claimName)` throughout the entire PR. Use `klog.KObj` when there is a claim object.